addon/components/tooltip-on-parent.js

```diff
@@ -11,16 +11,22 @@ export default Ember.Component.extend({
     return Ember.String.htmlSafe('display:none;');
   }),
 
-  registerOnParent: on('didInsertElement', function() {
-    const parentView = this.get('parentView');
+  init(...args) {
+    this._super(args);
 
-    if (parentView.renderTooltip) {
-      parentView.renderTooltip(this);
-    } else {
-      warn('No renderTooltip method found on the parent view of the {{tooltip-on-parent}} component');
-    }
+    run.schedule('afterRender', () => {
+      const parentView = this.get('parentView');
+      const componentNotInDOM = get(parentView, 'tagName') ? false : true;
+      if(componentNotInDOM) {
```

Comment:
My point being that returning boolean values from a ternary operation seems a little redundant.